src/app/Main/Body/QueryPage/Container-Choose.component.tsx

```diff
@@ -3,15 +3,20 @@ import { ErrorPagePageNotFoundComponent } from "../ErrorPage/ErrorPage-PageNotFo
 import { QueryPageComponent } from "./QueryPage.component";
 import { LoadingProcessComponent } from "../../../Shared/LoadingProcess.component";
 
-export function ChooseComponent(props: {
-    status: {
-        isolateStatus: number;
-        filterStatus: number;
-    } | undefined;
+export function ContainerChooseComponent(props: {
+    status:
+        | {

```

Comment:
? Why is there a pipe | before the {

Reply:
I don't know - it came with the "prettier - code formatter"

Reply:
ok, this is allowed in typescript. The formatter makes this so that types like:
type foo =
| foo1
| foo2
| foo3
...
looks prettier. It's the same like:
const foo = {
foo1,
foo2,
foo3, (here is a comma at the end)
}
I'm not a fan of that 'convention' because it looks like if someone forgot a type or property.